.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,13 @@
 <!-- You can view Contributing.MD for a detailed description of the pull request process. -->
 
 ## What Does This PR Do
-<!-- Include a small to medium description of what your PR changes. Document every changes or this may delay review or even discourage maintainers from merging your PR! -->
+<!-- Include a small to medium description of what your PR changes. Document all changes, not doing this may delay reviews or even discourage maintainers from merging your PR! -->
```

Comment:
```suggestion
<!-- Include a small to medium description of what your PR changes. Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
```